src/pystatis/table.py

```diff
@@ -34,10 +36,20 @@ def get_data(self, area: str = "all", prettify: bool = True, **kwargs):
             area (str, optional): Area to search for the object in GENESIS-Online. Defaults to "all".
             prettify (bool, optional): Reformats the table into a readable format. Defaults to True.
         """
-        params = {"name": self.name, "area": area, "format": "ffcsv"}
+        params = {
+            "name": self.name,
+            "area": area,
+            "format": "ffcsv",
+            "language": "de",
+        }
 
         params |= kwargs
 
+        if params["language"] not in ["de", "en"]:
+            raise NotImplementedError(
```

Comment:
this error type is normally not used for cases like this, it is really meant to signal that code needs to be implemented, but here it is more about a `ValueError` because the parameter has the wrong value as it should be one of "de" or "en"

Reply:
maybe we should even implement our own ErrorType here to signal that this is a `QueryParameterError`